plugins/ingest-attachment/src/test/java/org/elasticsearch/ingest/attachment/AttachmentProcessorTests.java

```diff
@@ -55,7 +55,7 @@
     @Before
     public void createStandardProcessor() {
         processor = new AttachmentProcessor(randomAlphaOfLength(10), null, "source_field",
-            "target_field", EnumSet.allOf(AttachmentProcessor.Property.class), 10000, false, null);
+            "target_field", EnumSet.allOf(AttachmentProcessor.Property.class), 10000, false, null, "resource_name");
```

Comment:
Let's leave this unspecified so the existing behavior of the processor without a `resource_name` is still verified.

```suggestion
            "target_field", EnumSet.allOf(AttachmentProcessor.Property.class), 10000, false, null, null);
```